onedal/linear_model/tests/test_logistic_regression.py

```diff
@@ -89,6 +89,7 @@ def test_csr(queue, dtype, dims):
         model_sp.fit(X_sp, y, queue=queue)
         pred_sp = model_sp.predict(X_sp, queue=queue)
 
-        assert_allclose(pred, pred_sp)
-        assert_allclose(model.coef_, model_sp.coef_)
-        assert_allclose(model.intercept_, model_sp.intercept_)
+        rtol = 2e-4 if dtype == np.float32 else 1e-5
```

Comment:
Are there any fails at float64 level? Because if 1e-7 works without fail for that, then this value would be ideal

Reply:
Anatoly said there are fails if the tolerance is less when I asked him. probably worth separate investigating

Reply:
There were some sporadic failures with difference 2e-6

Reply:
Seems it is not enough, the tolerance should be increased to 2e-4 for both floating point types. Definitely would need to further investigate it